deployment/terraform/live/vars.tf

```diff
@@ -12,7 +12,7 @@ variable "availability_zone_b" {
 }
 
 variable "identity_pool_name" {
-  description = "Cognito federated identity pool name"
+  description = "Cognito federated identity pool name. Constraints: must be unique within AWS account, [\\w]+"
```

Comment:
Identity pool names do not have to be unique within the AWS account (though it might be useful for easier pool management). Identity pool ID is used for connecting to the correct pool.